finagle-http/src/test/scala/com/twitter/finagle/http/StatusTest.scala

```diff
@@ -15,4 +16,45 @@ class StatusTest extends FunSuite {
     assert(Status(599).reason == "Server Error")
     assert(Status(601).reason == "Unknown Status")
   }
+
+  test("Pattern match usability check") {
+    null match {
```

Comment:
`null` isn't a status, therefore, it shouldn't be matched by any of the category matchers.